blockchain_exploration/currencies.py

```diff
@@ -52,5 +52,19 @@ class CryptoCurrency(DefaultDict):
             "native_token": True,
             "display_text": "sui",
         }
+    },
+    Network.TON: {
+        "ton": {
+            "symbol": "TON",
+            "code": "ton",
+            "native_token": True,
+            "display_text": "ton",
+        },
+        "scor": {
+            "symbol": "SCOR",
+            "code": "scor",
+            "native_token": False,
+            "display_text": "scor",
```

Comment:
Be careful here - if this is really display text, then it needs to be: `$SCOR` or `SCOR` as per the style guide (https://docs.google.com/document/d/1pmVol2Tmsl2uzSLz-Es4sLnozDPkDsgiDxhvQKJpaW4/edit?tab=t.0).

If this is just an internal keyword (as I suspect it is) then maybe add a small comment here.